City-of-Helsinki/drupal-hdbt
# UHF-2777 reduce heading font size

# Fixes [UHF-2751](https://helsinkisolutionoffice.atlassian.net/browse/UHF-2751) and [UHF-2983](https://helsinkisolutionoffice.atlassian.net/browse/UHF-2983)

Sorry for the confusion with the branch name, UHF-2777 is tech debt epic. 🤦 

## What was done
* Font tokens were written to scss variable map
* All font-sizes were converted to use font tokens from design
  * Originally only headings were meant to be touched and the rest in another ticket, but it was found out to be impossible to do in parts
  * Luckily the work went faster than expected
* font-size() mixin was removed
* Unused styles were removed
* Variable files were renamed to end in ..`_variable` for clarity

## How to test

* Get branch
* Check that fonts are rendered in a sane way
* Compare to production heading sizes, notice that most of them are smaller as promised
* Check new coding conventions that they're readable
